src/uci.cpp

```diff
@@ -164,7 +164,7 @@ namespace {
         }
         else if (token == "setoption")  setoption(is);
         else if (token == "position")   position(pos, is, states);
-        else if (token == "ucinewgame") Search::clear();
+        else if (token == "ucinewgame") { Search::clear(); elapsed = now(); } // Search::clear() may take some while
```

Comment:
```cpp
// start clock before search
```

Reply:
Timepoint doesn't need to be initialized at that point, if it is done in ucinewgame.

Reply:
Unfortunately, we then are running into an [-Wmaybe-uninitialized] warning.
```
uci.cpp:170:21: warning: ‘elapsed’ may be used uninitialized in this function [-Wmaybe-uninitialized]
     elapsed = now() - elapsed + 1; // Ensure positivity to avoid a 'divide by zero'
                     ^
uci.cpp:151:15: note: ‘elapsed’ was declared here
     TimePoint elapsed;
```

Reply:
Ok, so leave the init. It would be indeed uninit if there are no fens in the list.